README.md

```diff
@@ -9,15 +9,26 @@ The sole focus of the Fastblock feature is to restrict the loading of trackers.
 ## Development
 
 You must run the study with [Firefox
-Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly) or
-[Developer
-Edition](https://www.mozilla.org/firefox/developer/).
+Nightly](https://www.mozilla.org/en-US/firefox/channel/desktop/#nightly)
 
 See [Getting
 Started](https://github.com/mozilla/FastBlockShield/blob/master/docs/DEV.md#getting-started) for instructions to install, run, lint, and build the add-on.
 
 You should be able to `npm start -- -f Nightly`
 
+### Tests
+
+We currently have functional tests, you can find them under `test/functional/`.
+Please test your new code and make sure to run the tests before committing.
+
+To run the tests, use
+
+```shell
```

Comment:
add `npm run build` as an instruction - need to ensure there is something in the dist file